src/intl/en.json

```diff
@@ -22,6 +22,267 @@
+    "profile": "Profile",
+    "profile-info": " Tell us a little bit about yourself - whatever you think we should know.",
+    "expertise": "Area of expertise",
+    "expertise-info": "What is unique about your skill set? What kinds of problems do you enjoy solving?",
+    "why": "Why Ethereum?",
+    "proj-dev": "What's a project or development that you were excited about recently?",
+    "prev-work": "Previous work",
+    "link-2-work": " Please provide links to published code, research, or other documentation of what you're working on.",
+    "questions": "What questions can we answer for you?",
+    "country": "What country are you located in?",
+    "city": "What city are you located in?",
+    "support": "How did you hear about Ecosystem Support?",
+    "support-info":"Did anyone recommend that you contact Ecosystem Support? If so, who?",
+    "subscribe": "Subscribe to the ESP Newsletter? You'll hear from us every few weeks, and we'll only ever contact you with ESP news.",
+    "button": "Submit"
+  }
+},
+  "page-project": {
+    "title": "Specific Project",
+    "project": {
+      "h1": "Specific Project",
+      "description": "Tell us a bit about your project goals, motivations, needs, challenges, and any other information you feel is pertinent for us to know. You'll hear back from someone on the Ecosystem Support team very soon! We only collect the following information submitted below and will not use or share for any purposes other than evaluation.",
+      "proj-name": "Project name",
+      "proj-desc": "Project description",
+      "proj-success": "What are you working on? What does success look like for your project? (2-3 short paragraphs)",
+      "profile": "Team profile",
+      "team": "Who are the members of your team? Please write or link to bios.",
+      "impact": "Impact",
+      "proj-impact": "What key problem or need do you hope to address? How will your work impact the larger Ethereum ecosystem? Who will benefit from your work?",
+      "challenges": "Needs and Challenges",
+      "challenges-info": "What are the most significant obstacles facing your project or your team right now? What are some of your most pressing needs?",
+      "links": "Please provide links to published code, research, or other documentation of what you're working on.",
+      "contact-person": "Contact person",
+      "contact-email": "Contact email",    
+      "contact-p-country": "Where is your contact person based (country)?",
+      "contact-p-city": "Where is your contact person based (city)?",
+      "support": "How did you hear about Ecosystem Support?",
+      "support-p": "Did anyone recommend that you contact Ecosystem Support? If so, who?",
+      "subscribe": "Subscribe to the ESP Newsletter? You'll hear from us every few weeks, and we'll only ever contact you with ESP news.",
+      "button": "Submit"
+    }
+  },
+  "page-404": {
+    "title": "Page not found",
+    "404": {
```

Comment:
Just a note on this - there's really no need for this additional nesting. I think encapsulating the copy with `page-404` key is fine.